lowRISC/sonata-system
# Migrated to OpenTitan's UART and added UART DPI

To test run:

```sh
make -C sw/cpp/cheri_sanity uart_check.elf
fusesoc --cores-root=. run --target=sim --tool=verilator \
    --setup --build lowrisc:sonata:system
./build/lowrisc_sonata_system_0/sim-verilator/Vsonata_system \
    --meminit=ram,./sw/cpp/cheri_sanity/uart_check.elf
```

and you should see `hi` in the `uart0.log` file.
